sys/netpfil/pf/if_pfsync.c

```diff
@@ -1442,17 +1570,41 @@ pfsyncioctl(struct ifnet *ifp, u_long cmd, caddr_t data)
 			if_rele(sc->sc_sync_if);
 		sc->sc_sync_if = sifp;
 
-		ip = &sc->sc_template;
-		bzero(ip, sizeof(*ip));
-		ip->ip_v = IPVERSION;
-		ip->ip_hl = sizeof(sc->sc_template) >> 2;
-		ip->ip_tos = IPTOS_LOWDELAY;
-		/* len and id are set later. */
-		ip->ip_off = htons(IP_DF);
-		ip->ip_ttl = PFSYNC_DFLTTL;
-		ip->ip_p = IPPROTO_PFSYNC;
-		ip->ip_src.s_addr = INADDR_ANY;
-		ip->ip_dst.s_addr = sc->sc_sync_peer.s_addr;
+		printf("pfsync: Before template. AF: %d\n", sc->sc_sync_peer.sa.sa_family);
+		switch (sc->sc_sync_peer.sa.sa_family) {
+		case AF_INET: {
```

Comment:
Please add `ifdef …` for address families.

Reply:
Done.